object_database/web/cells/cells.py

```diff
@@ -1958,14 +2053,25 @@ def recalculate(self):
 
 
 class Popover(Cell):
+    # TODO: Does title actually need to be a cell here? What about detail?
+    # What is the purpose of the sortAs method here and why are we using
+    # it on the title cell?
```

Comment:
SortAs determines how any particular cell would sort in a Table. We have to pick something...